device/mellanox/x86_64-mlnx_msn2700-r0/platform.json

```diff
@@ -0,0 +1,484 @@
+{
+    "chassis": {
+        "name": "Mellanox-SN2700",
+        "model": "FIXME",
+        "serial": "FIXME",
+        "components": [],
+        "modules": [],
+        "fans": [
+            {
+                "name": "Fan 1",
+                "direction": "intake",
```

Comment:
currently FAN direction only available on Mellanox Spectrum 2 and Spectrum 3 platform, for 2700 it's should be 'N/A'

Reply:
fixed